src/Files/UserControls/NavigationToolbar.xaml

```diff
@@ -562,6 +562,28 @@
                         Value="{x:Bind OngoingTasksViewModel.InfoBadgeValue, Mode=OneWay}" />
                 </Grid>
 
+                <Button
+                    x:Name="UpdateButton"
+                    x:Load="{x:Bind ViewModel.UpdateSettingsService.IsUpdateAvailable, Mode=OneWay}"
+                    Width="36"
+                    Height="32"
+                    AccessKey="U"
+                    AutomationProperties.AutomationId="UpdateButton"
+                    AutomationProperties.Name="{helpers:ResourceString Name=UpdateAvailable}"
+                    Background="Transparent"
+                    Command="{x:Bind ViewModel.UpdateCommand, Mode=OneWay}"
+                    IsEnabled="{x:Bind ViewModel.UpdateSettingsService.IsUpdating, Mode=OneWay, Converter={StaticResource BoolNegationConverter}}"
+                    Style="{StaticResource ToolBarButtonStyle}"
+                    ToolTipService.ToolTip="{helpers:ResourceString Name=UpdateAvailable}"
+                    HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch">
+                    <Grid>
+                        <FontIcon FontSize="14" Glyph="&#xE896;" />
+                        <muxc:InfoBadge
```

Comment:
Can you remove the info badge? Showing the icon itself should be enough of an indicator since it's usually hidden.